packages/messaging/src/testing/fakes/firebase-dependencies.ts

```diff
@@ -47,24 +53,26 @@ export function makeFakeApp(options: FirebaseOptions = {}): FirebaseApp {
     options,
     automaticDataCollectionEnabled: true,
     delete: async () => {},
-    messaging: null as any,
-    installations: null as any
+    messaging: (() => null as unknown) as FirebaseApp['messaging'],
+    installations: () => getFakeInstallations()
   };
 }
 
-export function makeFakeInstallations(): FirebaseInstallations {
+function getFakeInstallations(): FirebaseInstallations {
   return {
     getId: async () => 'FID',
     getToken: async () => 'authToken',
     delete: async () => undefined
   };
 }
 
-export function makeFakeAnalyticsProvider(): Provider<
-  FirebaseAnalyticsInternalName
-> {
-  return new Provider<FirebaseAnalyticsInternalName>(
-    'analytics-internal',
-    new ComponentContainer('test')
-  );
+function getFakeAnalyticsProvider(): Provider<FirebaseAnalyticsInternalName> {
+  const analytics: FirebaseAnalyticsInternal = {
+    logEvent: () => {}
```

Comment:
drop ":" and "=>", just "logEvent() {}"? :)

Reply:
Sure.